scripts/babel/transform-prevent-infinite-loops.js

```diff
@@ -13,7 +13,7 @@
 // This should be reasonable for all loops in the source.
 // Note that if the numbers are too large, the tests will take too long to fail
 // for this to be useful (each individual test case might hit an infinite loop).
-const MAX_SOURCE_ITERATIONS = 1500;
+const MAX_SOURCE_ITERATIONS = 5000;
```

Comment:
Why did you have to bump this?

Reply:
It's in the commit comment but escapeTextForBrowser is a tight loop for escaping and it easily gets up there with escaping stack traces.

We should probably find a way to disable this for hot loops.